### PR TITLE
Hotfix/v1.0.3: Updated the init.sh,config.sh and Readme

### PR DESCRIPTION
**Pull request by waji46**
Issue:
        Title:Hotfix for init.sh,config.sh and Readme #33 

     
        [PS-1]
         - Fixed an issue related to keypair, updated the init.sh and config.sh file.
         - Update the readme file.

### CODE REVIEW

**Comment by faheem-khau9** [comment]
:robot: OpenAI








---

### Chat with 🤖 OpenAI Bot (`@openai`)
- Reply on review comments left by this bot to ask follow-up questions. A review comment is a comment on a diff or a file.
- Invite the bot into a review comment chain by tagging `@openai` in a reply.

### Code suggestions
- The bot may make code suggestions, but please review them carefully before committing since the line number ranges may be misaligned. 
- You can edit the comment made by the bot and manually tweak the suggestion if it is slightly off.

### Ignoring further reviews
- Type `@openai: ignore` anywhere in the PR description to ignore further reviews from the bot.

---






<details>
<summary>Files not summarized due to errors (3)</summary>

### Failed to summarize

* infrastructure/init.sh (nothing obtained from openai)
* infrastructure/config.sh (nothing obtained from openai)
* README.md (nothing obtained from openai)

</details>


---
In the recent run, only the files that changed from the `base` of the PR and between `bd9b8c185541883e784d505ca2e658009e5147be` and `7264bd6024c277e0c81b7c4de03f0a13462b1e1c` commits were reviewed.

<details>
<summary>Files not reviewed due to errors in the recent run (3)</summary>

### Failed to review in the last run

* infrastructure/config.sh (no response)
* infrastructure/init.sh (no response)
* README.md (no response)

</details>

File: README.md [hunk 1]
```diff
@@ -91,18 +91,16 @@ Check the below video for a quick demo of XC3.
     ```
     cd xc3/infrastructure
     bash init.sh
+    chmod 400 "keypair.pem"
```
Comment (samanXgrid) [inline README.md:94]: Add this command before sudo ssh command not here

File: infrastructure/init.sh [hunk 2]
```diff
@@ -102,7 +106,7 @@ fi
 
 
 # Create EC2 Key-Pair that will be used to ssh into EC2 instances spin up by terraform module
-if aws ec2 create-key-pair --key-name "${project}-key" --tag-specifications "ResourceType=key-pair,Tags=[{Key=Project,Value=${project}},{Key=Owner,Value=${owner_email}},{Key=Creator,Value=${creator_email}}]" --query 'KeyMaterial' --output text > "${project}"-key.pem --region="${aws_region}" ; then
+if aws ec2 create-key-pair --key-name "${namespace}-key" --tag-specifications "ResourceType=key-pair,Tags=[{Key=Project,Value=${project}},{Key=Owner,Value=${owner_email}},{Key=Creator,Value=${creator_email}}]" --query 'KeyMaterial' --output text > "${project}"-key.pem --region="${aws_region}" ; then
```
Comment (samanXgrid) [inline infrastructure/init.sh:109]: update same command in init-delete.sh as well

File: README.md (hunk 1 above)
Comment (waji46) [reply README.md]: Feedback addressed.

File: infrastructure/init.sh (hunk 2 above)
Comment (waji46) [reply infrastructure/init.sh:109]: Updated teh init_delete.sh file

**Review by samanXgrid — APPROVED** [review]
LGTM

**Review by yaseenzafar123 — APPROVED** [review]
LGTM

**Review by muhdsaad — APPROVED** [review]
LGTM

**Review by faheem-khau9 — APPROVED** [review]
LGTM

**Pull request merged**